### PR TITLE
o3-input-text for whitelabel

### PR DESCRIPTION
**Pull request by itsdeyan**
## Describe your changes
added tokens for the input text component for whitelabel
## Checklist before requesting a review

- [ ] I have applied `percy` label for o-[COMPONENT] or `chromatic` label for o3-[COMPONENT] on my PR before merging and after review. Find more details in [CONTRIBUTING.md](https://github.com/Financial-Times/origami/blob/main/CONTRIBUTING.md#pull-requests-and-visual-regression-tests)
- [ ] If it is a new feature, I have added thorough tests.
- [ ] I have updated relevant docs.
- [ ] I have updated relevant env variables in Doppler.

### CODE REVIEW

File: apps/dictionary/tokens/whitelabel/base/color.json [hunk 1]
```diff
@@ -18,6 +18,10 @@
           "value": "#c00",
           "type": "color"
         },
+        "crimson-10": {
```
Comment (frshwtr) [inline apps/dictionary/tokens/whitelabel/base/color.json:21]: This value isn't used, do we need it here?

File: apps/dictionary/tokens/$metadata.json [hunk 2]
```diff
@@ -42,6 +42,7 @@
     "whitelabel/components/o3-tooltip",
     "whitelabel/components/o3-typography",
     "whitelabel/components/o3-editorial-typography",
+    "whitelabel/components/o3-input",
```
Comment (akomiqaia) [inline apps/dictionary/tokens/$metadata.json:45]: are we planing to name this component o3-input? or does it need to be under o3-form?

File: apps/dictionary/tokens/whitelabel/base/color.json (hunk 1 above)
Comment (akomiqaia) [reply apps/dictionary/tokens/whitelabel/base/color.json:21]: Presumably, it will be used in future? or maybe I am misunderstanding your comment.

**Comment by notlee** [comment]
@akomiqaia @frshwtr, is this PR still needed or have tokens been added elsewhere?

**Comment by akomiqaia** [comment]
> @akomiqaia @frshwtr, is this PR still needed or have tokens been added elsewhere?

This PR can be closed. We don't have a component called `o3-input`. We created `o3-form` component instead.

**Pull request closed without merging**